obc.py

```diff
@@ -33,7 +38,12 @@ def stop(self):
         # stop apps
         self.imageService.stop()
         self.timelapse.stop()
+<<<<<<< HEAD
```

Comment:
Similar thing here as above. Please remove `<<<<<<< HEAD`, `=======` and `>>>>>>> 4fc1f82be6b1373c222d146900e064766842638e`